php/src/Product.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace GildedRose;
+
+interface Product
+{
+    public function updateQuality(object $item): void;
```

Comment:
```suggestion
    public function updateQuality(Item $item): void;
```
應準確要求傳入的是該 Item 型別，而非任意 object